src/main/resources/application.properties

```diff
@@ -15,3 +15,10 @@ spring.main.allow-bean-definition-overriding=true
 # Internationalization
 spring.messages.basename=messages
 spring.messages.encoding=UTF-8
+
+# SSL Encryption
+server.ssl.enabled=true
+server.ssl.key-store-type=PKCS12
+server.ssl.key-store=classpath:keystore.p12
+server.ssl.key-store-password=badpass
+server.ssl.key-alias=tomcat
```

Comment:
Correct me if I'm wrong since there're multiple options for securing a Spring application, but here's the way I learned:

- **Do** include **multiple** application **properties** depending on state (dev, prod, etc.) - Contact Daniel or someone from the project board for further information
- **Don't** track and publish properties containing **sensible information** in Git! - This is a security issue.